packages/editor/src/editor/sidebar/fields/InputFieldWithBrowser.tsx

```diff
@@ -15,16 +15,40 @@ export const InputFieldWithBrowser = ({
   options
 }: InputFieldProps & { browsers: Browser[]; options?: TextBrowserFieldOptions }) => {
   const [open, setOpen] = useState(false);
+  const [autoselectInput, setAutoselectInput] = useState(false);
+  const inputRef = useRef<HTMLInputElement>(null);
   const attrBrowser = useAttributeBrowser(options?.onlyAttributes, options?.onlyTypesOf);
   const logicBrowser = useLogicBrowser();
 
   const activeBrowsers = [...(browsers.includes('ATTRIBUTE') ? [attrBrowser] : []), ...(browsers.includes('LOGIC') ? [logicBrowser] : [])];
 
+  useEffect(() => {
+    if (autoselectInput && inputRef.current) {
+      setTimeout(() => {
+        if (autoselectInput && inputRef.current) {
+          const inputElement = inputRef.current;
+          const match = value.match(/\(([^)]+)\)/);
+          if (match) {
+            const textToSelect = match[1];
+            const startIndex = value.indexOf(textToSelect);
+            const endIndex = startIndex + textToSelect.length;
+            inputElement.focus();
+            inputElement.setSelectionRange(startIndex, endIndex);
+          }
+        }
+      }, 200);
```

Comment:
second, a timout needs to be cleaned in a useeffect via return.

Reply:
third, instead of this autoselect + useeffect stuff, why not directly set this timeout from the apply browser function?

Reply:
fourth, I'm not really happy with this timeout stuff... This maybe works on your PC, but is it also working on a slower machine?
maybe it is better to simple not do this until we have also a monaco editor for this inputs...